server/views/pages/visit/summary.njk

```diff
@@ -20,6 +22,39 @@
 
       <h1 class="govuk-heading-l">{{ pageHeaderTitle }}</h1>
 
+       {% if visit.visitStatus === "CANCELLED" %}
+        {% set status = visit.outcomeStatus.split("_") %}
+            {%- for visitNote in visit.visitNotes %}
+              {% if visitNote.type == "VISIT_OUTCOMES" %}
+
+                {% set cancelledBy %}
+                  {% if status[0] === 'ADMINISTRATIVE' %}
+                    {{ 'because of an administrative error.' }}
```

Comment:
Cleverly split out! Just wondering though about the text ("because of an administrative error") used? Couldn't see it on the ticket so just checking if Ed/Phoebe came up with this. If not, might be worth checking with them if they're happy with what the text will end up being under the [four possible cases](https://github.com/ministryofjustice/book-a-prison-visit-staff-ui/blob/main/server/constants/visitCancellationReasons.ts).

Only suggesting this as I wondered if something like 'administrative reasons' might be a bit more generic than 'administrative error' 🤷‍♂️

Reply:
I noticed that the wording would have been 'by the administrative error' if I implemented this case like I have the others, was hoping to get some feedback on what was best for this case, but knew we would deal with it differently